course/src/main/java/org/openedx/course/presentation/container/CourseContainerViewModel.kt

```diff
@@ -150,6 +151,8 @@ class CourseContainerViewModel(
     val courseDetails: CourseEnrollmentDetails?
         get() = _courseDetails
 
+    var courseContainerTabs = MutableStateFlow<List<CourseContainerTab>>(CourseContainerTab.entries)
```

Comment:
IMO, it's better to introduce the get function cuz `courseContainerTabs` doesn’t change over time.